src/components/fields/field-input/field-input.component.tsx

```diff
@@ -29,7 +29,7 @@ export const InputField = ({
       <label className="fdk-form-label w-100 fdk-text-strong position-relative" htmlFor={field.name}>
         {showLabel ? label : null}
         {!!language && !isOnlyOneSelectedLanguage && <span className="language-indicator">{language}</span>}
-        <input {...field} type={type} className="form-control" autoComplete="off" />
+        <input {...field} type={type} onBlur={() => {}} className="form-control" autoComplete="off" />
```

Comment:
Jeg skjønner ikke denne. Ønsker du å unngå validering på onBlur? Jeg mener Formik har en egen setting for å styre om validering skjer på onChange, onBlur eller begge deler. Jeg tror det er en bedre måte å gjøre det på enn en tom funksjon på hvert felt.

Reply:
Ok, lagt til på Formik config.